src/CMakeLists.txt

```diff
@@ -91,7 +91,7 @@ target_link_libraries(lximage-qt
     ${QT_LIBRARIES}
     ${EXIF_LIBRARIES}
     ${X11_LIBRARIES}
-    ${XFIXES_LIBRARIES}
+    ${XFIXES_LDFLAGS}
```

Comment:
IMO everywhere (pkg-config's configurations) should be used \*_LDFLAGS instead of \*_LIBRARIES ...

Reply:
Huh. I haven’t encountered problems with other *_LIBRARIES inclusions (except POLKIT_AGENT_LIBRARIES).